test/cadet_web/controllers/grading_controller_test.exs

```diff
@@ -440,6 +547,43 @@ defmodule CadetWeb.GradingControllerTest do
     end
   end
 
+  describe "GET /group, staff" do
+    @tag authenticate: :staff
```

Comment:
Please do add one for each of the roles (missing `:student` and `:admin`

Reply:
Okay! Added 2 more test cases